### DISCUSSION TITLE
Repositories List Default Sort Order Selection

### DISCUSSION THREAD

**Discussion by ArnieGA** [community/community#64964] [Repositories]
### Select Topic Area

Product Feedback

### Body

Hello 😀

The current way to sort repositories in the repositories list (Repositories tab) is very annoying. `Last updated` is selected by default, which causes unexpected behaviors when navigating back and forth.
<img width="454" alt="image" src="https://github.com/community/community/assets/13170077/6e22caa6-13aa-49d0-96b9-b0c81c5d7f62">

For example, if I change my selection to `Name`, it might or might not persist while navigating back and forth tabs or repo code. This is very frustrating and very annoying --one expects it to persist when navigating back to the Repositories tab, but it doesn't. So I have to keep selecting `Name` each time it goes back to `Last updated`.

I was thinking that maybe adding the ability to select one of the choices as the default choice could help?
Or possibly just making the selection persist?

**Comment by silnith** [comment]
Two years and not even an acknowledgement?
